src/utils/accessibility/accessibility.js

```diff
@@ -1,22 +1,36 @@
+// 1pt = 1.333px according to http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#visual-audio-contrast-contrast-73-head
+const PIXELS_IN_POINT = 1.333;
+
 // http://www.w3.org/TR/2008/REC-WCAG20-20081211/#larger-scaledef
 function isLargeScale(fontSize, isFontBold) {
   return isFontBold ? (fontSize >= 14) : (fontSize >= 18);
 }
 
-function accessibleContrast(accessibilityLevel, fontSize, isFontBold) {
+function fontSizeInPt(fontSize, fontSizeUnit) {
+  // http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#visual-audio-contrast-contrast-73-head
+  return (fontSizeUnit === 'pt') ? fontSize : Math.round(fontSize / PIXELS_IN_POINT);
+}
+
+function fontSizeInPx(fontSize, fontSizeUnit) {
+  // http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#visual-audio-contrast-contrast-73-head
```

Comment:
Maybe we can remove this comment, and the one in `fontSizeInPt()`, since we already have it at the top of the file?